File: VB/WebApplication1/Models/DataModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace WebApplication1.Models
	Public Class Customer
		Public Property CustomerID() As Integer
		Public Property CustomerName() As String
		Public Property Licenses() As Object
	End Class
	Public Class License
		Public Property LicenseID() As Integer
		Public Property LicenseName() As String
		Public Property ParentLicenseID() As Integer
	End Class
End Namespace

File: CS/WebApplication1/Models/DataModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models {
    public class Customer {
        public int CustomerID { get; set; }
        public string CustomerName { get; set; }
        public object Licenses { get; set; }
    }
    public class License {
        public int LicenseID { get; set; }
        public string LicenseName { get; set; }
        public int ParentLicenseID { get; set; }
    }
}
